### DISCUSSION TITLE
Dataset refinement

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/dpgen#1599] [Q&A]
Dear all,

I have generated a large DFT dataset using VASP calculations, and I would like to reduce the number of configurations.
So, I mean, I'd like to exclude the unphysical or unstable configurations with a large error. Can I archive this problem with DP-GEN?

**Comment by njzjz** [comment]
You can use `dpgen simplify` and give a proper `model_devi_f_trust_hi` or `model_devi_e_trust_hi`.

However, I am wondering why not exclude high-energy points directly considering you have already had DFT energies.
